forms/consent/BSWH_Consent_V0.02.html

```diff
@@ -1195,7 +1195,7 @@ <h1 style="padding-top: 10pt;padding-left: 20pt;text-indent: 0pt;text-align: lef
             class="s10" target="_blank">You may also contact the Principal Investigator, Nicolas Wentzensen, M.D., Ph.D., M.S. 
             at </a>ConnectPI@nih.gov or 240-276-7150.</p>
     <p style="padding-top: 5pt;padding-left: 20pt;text-indent: 0pt;text-align: left;">For questions about your rights
-        while in this study, call the NIH Institutional Review Board at 301-402-3713.</p>
+        while in this study, call the NIH Institutional Review Board at 301-402-3713. </p> <br />
```

Comment:
`<br>` is preferred, according to [Standard](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element).